### PR TITLE
Hide fronts banner slots when a pageskin is running

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Hide fronts banner slots when a pageskin is running

## Why?
Due to the pageskin, the slots display blank spaces, we can remove the slots when a pageskin is running server side.

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/guardian/dotcom-rendering/assets/1731150/879d0dce-8564-470c-85d3-f1efa5d46b83
[after]: https://github.com/guardian/dotcom-rendering/assets/1731150/e6610c78-65cd-433d-bade-bbcdb0ebc536

### CODE REVIEW

**Comment by domlander** [comment]
It looks like we will serve MPU's when in the section-banner test and a pageskin is displayed. Is this intentional?

File: dotcom-rendering/src/layouts/FrontLayout.tsx [hunk 1]
```diff
@@ -99,7 +99,10 @@ export const decideAdSlot = (
 		collectionCount > minContainers &&
 		index === getMerchHighPosition(collectionCount)
 	) {
-		if (isInNetworkFrontsBannerTest || isInSectionFrontsBannerTest) {
+		if (
+			(isInNetworkFrontsBannerTest || isInSectionFrontsBannerTest) &&
+			!hasPageSkin
```
Comment (domlander) [inline dotcom-rendering/src/layouts/FrontLayout.tsx:104]: I think this will hit the `else` condition and display a `merchandising-high` slot when there is a pageskin. Nesting the `hasPageSkin` check to inside this if condition and returning null if so, should work

Comment (Jakeii) [reply dotcom-rendering/src/layouts/FrontLayout.tsx]: Thanks, actually I noticed we don't need any changes here at all 🤦‍♂️ This logic is hard to follow defo needs a refactor after the test!

Comment (domlander) [reply dotcom-rendering/src/layouts/FrontLayout.tsx]: Yeah, agreed that it's confusing. It's tricky because it's temporary code that should exist for the duration of the two tests. Now, with the likely delay of the go-live date, this code might exist for a bit longer than initially anticipated.

**Review by domlander — APPROVED** [review]
